.github/workflows/semgrep.yml

```diff
@@ -0,0 +1,19 @@
+name: Semgrep
+on:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  semgrep:
+    name: Run Semgrep 
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    container:
+      # A Docker image with Semgrep installed. Do not change this.
+      image: returntocorp/semgrep
+    #if: (github.actor != dependabot[bot])
```

Comment:
can we remove this line if its not needed?

Reply:
Okay, I'll remove it.